internal/cf/client.go

```diff
@@ -210,5 +254,257 @@ func NewSpaceHealthChecker(spaceGuid string, url string, username string, passwo
 		}
 	}
 
+	if config.IsResourceCacheEnabled && client.resourceCache == nil {
```

Comment:
@TheBigElmo 
It assigns the `resourceCache` created by the organization unit client and space client and will use the `spaceCache` populated by the organization unit client in the `cf/health.go`.

Currently the `health.go` uses the spaceClient to call the CF API. Is this OK to use the cache populated by the organization client?